apps/condo/domains/marketplace/schema/Invoice.test.js

```diff
@@ -1101,6 +1101,89 @@ describe('Invoice', () => {
+                invoiceRecipient: createTestRecipient(),
+            })
+
+            const unitType = FLAT_UNIT_TYPE
+            const unitName = faker.lorem.word()
+
+            const residentClient = await makeClientWithResidentUser()
+            const [resident] = await registerResidentByTestClient(
+                residentClient,
+                {
+                    address: client.property.address,
+                    addressMeta: client.property.addressMeta,
+                    unitType,
+                    unitName,
+                })
+
+            const [ticket] = await createTestTicket(client, client.organization, client.property, {
+                isPayable: true,
+                unitType,
+                unitName,
+                clientName: null,
+                clientPhone: null,
+                contact: null,
+                client: { connect: { id: residentClient.user.id } },
+            })
+            const [createdInvoice] = await createTestInvoice(client, client.organization, {
+                property: { connect: { id: client.property.id } },
+                unitType,
+                unitName,
+                status: INVOICE_STATUS_DRAFT,
+                client: { connect: { id: residentClient.user.id } },
+                ticket: { connect: { id: ticket.id } },
+                clientName: null,
+                clientPhone: null,
+                contact: null,
+            })
+
+            const invoice = await Invoice.update(adminClient, createdInvoice.id, {
+                status: INVOICE_STATUS_PUBLISHED,
+                dv: 1,
+                sender: { dv: 1, fingerprint: 'tests' },
+            })
+
+            let messages
+            await waitFor(async () => {
+                messages = await Message.getAll(adminClient, {
+                    user: { id: residentClient.user.id },
+                    type: MARKETPLACE_INVOICE_WITH_TICKET_PUBLISHED_MESSAGE_TYPE,
+                })
+                expect(messages).toHaveLength(1)
+            })
+
+
+            const [createdInvoice2] = await createTestInvoice(client, client.organization, {
+                property: { connect: { id: client.property.id } },
+                unitType,
+                unitName,
+                status: INVOICE_STATUS_DRAFT,
+                client: { connect: { id: residentClient.user.id } },
+                ticket: { connect: { id: ticket.id } },
+                clientName: null,
+                clientPhone: null,
+                contact: null,
+            })
+            await waitFor(async () => {
+                messages = await Message.getAll(adminClient, {
+                    user: { id: residentClient.user.id },
+                    type: MARKETPLACE_INVOICE_WITH_TICKET_PUBLISHED_MESSAGE_TYPE,
+                })
+                expect(messages).toHaveLength(1)
+            })
+
+            expect(messages[0].meta.data.invoiceId).toEqual(invoice.id)
```

Comment:
Need to be sure that `message[0]` here has the same id as the `messages[0]` above